redaxo/src/core/lang/de_de.lang

```diff
@@ -1,55 +1,34 @@
-lang = Deutsch
 htmllang = de
-yes = ja
-no = nein
```

Comment:
Werden verwendet: https://github.com/redaxo/redaxo/blob/3c39cca4ef605a8902c25647c89b16cee56abb71/redaxo/src/addons/structure/plugins/content/pages/templates.php#L574

Reply:
Wir müssen erst einmal entscheiden ob der Core weiterhin Keys enthalten soll, die er selber nicht nutzt.
Einige sind dabei, die scheinbar nur von AddOn (hier structure) verwendet werden. 

Das Tool parsed die Dateien vom Core nach den verwendeten Keys und löscht dann alle aus den lang Dateien die nicht genutzt werden (im Idealfall)

Reply:
wenn wir im core jetzt lang keys entfernen und in core addons verlagern wäre dass ein BC break. daher sollten wir solche allgemeinen keys nicht verschieben.. zusätzlich werden die sicher zum teil auch in addons verwendet die nicht in unserem repo sind